engine/swarm/how-swarm-mode-works/services.md

```diff
@@ -106,14 +106,14 @@ For a replicated service, you specify the number of identical tasks you want to
 run. For example, you decide to deploy an HTTP service with three replicas, each
 serving the same content.
 
-A global service is a service that runs one task on every node. There is no
+A global service is a service that runs one copy of the task on every node. There is no
```

Comment:
Don't think "one copy of the task" is correct here; as `task` in itself is already meaning "an instance of the service"

Reply:
Thanks for the clarification. I will decline the PR.